README.md

```diff
@@ -1,8 +1,21 @@
 # Cthulhu.jl [![CI](https://github.com/JuliaDebug/Cthulhu.jl/actions/workflows/CI.yml/badge.svg)](https://github.com/JuliaDebug/Cthulhu.jl/actions/workflows/CI.yml)
 
-*The slow descent into madness*
-
-:warning: The latest stable version is only compatible with Julia v1.8.5 and higher.
+> *The slow descent into madness*
+
+> [!warning]
+> This package relies on internal APIs of the Julia base compiler.
+> As such, its behavior may change depending on your Julia version.
+> For maintenance reasons, the latest version of Cthulhu (typically managed in the `master`
+> branch) does not guarantee compatibility with older Julia versions.
+> Versions of Cthulhu compatible with earlier Julia releases may be maintained in dedicated branches[^old-version-branch-ex].
+>
+> Please refer to the `Project.toml` file for details on supported Julia versions.
+> Generally, Julia's package manager automatically selects a compatible Cthulhu version for
+> your Julia installation.
+> If you simply want to use Cthulhu, running `pkg> add Cthulhu` will install an appropriate version.
```

Comment:
Did you mean for these `>` to be here?

Reply:
Yea, 
```md
> [!admonition-header]
> ...
```
is the syntax for showing admonition on GitHub markdown.

Reply:
For the `*The slow descent into madness*` too? This one seems outside of the admonition block.

Reply:
We didn't really need that part, but the sentence itself is more like a casual comment than documentation, so I just quoted it;)